### PR TITLE
FEATURE: Landing Page for Website Introduction/Summary

### PR DESCRIPTION
**Pull request by Domico-Carlo**
# Includes features of:
## - A landing page with brief description/pitch
image

## - A persistent navigation bar redirecting to different pages
image

### CODE REVIEW

**Comment by Rishi-Bidani** [comment]
The 2nd button says "Another button". Do we wanna fix?

File: frontend/src/pages/FAQPage.jsx
Comment (Alucardigan) [inline frontend/src/pages/FAQPage.jsx:1]: this file is not needed

Comment (Domico-Carlo) [reply frontend/src/pages/FAQPage.jsx:1]: right

**Review by Alucardigan — APPROVED** [review]
yeah this is fine. Approved

**Pull request merged**